core/src/scss/components/card/_card.scss

```diff
@@ -1,14 +1,28 @@
 @charset "UTF-8";
 
 //
-// Card
+// Card (Single Link)
```

Comment:
I don't think we want to change the name of this do we?

Reply:
I'm thinking that maybe we do at least in the documentation title since Kerri is introducing another type of cards (the link list cards) with multiple links (and no separate link buttons/CTA/image/headline links) so the logic is different on that one. It might be less confusing to make that a different component.

Reply:
![Screen Shot 2019-11-25 at 3 16 38 PM](https://user-images.githubusercontent.com/42749717/69586190-9e166400-0f96-11ea-836c-447900b62683.png)

Reply:
I don't see why this couldn't or shouldn't be the same component. We should have a sit-down session and hash out all the variants and our approach to keeping the template simple and not overly complicated.

Reply:
We may need to pivot the markup for v6 or v7.

Reply:
> I don't see why this couldn't or shouldn't be the same component. We should have a sit-down session and hash out all the variants and our approach to keeping the template simple and not overly complicated.

Yeah, I was thinking since we're redoing the site, this being one component is fine. With our current setup it's hard to demonstrate all the possibilities, but with the new site it would be much better. So, I'm fine with making this one component.

Reply:
Hmmm well, that depends. If we drop the KSS renderer we can customize our components page to whatever we want but will have to build something and if we keep the KSS renderer we will still have the same issues with rendering the previews. 

Something to think about before palooooooza.

Reply:
Perhaps this is a fit for 'collections' in the new system.